src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue14471.cs

```diff
@@ -1,4 +1,5 @@
-﻿using NUnit.Framework;
+﻿#if TEST_FAILS_ON_ANDROID // This test fails randomly, likely because the image source has disappeared sometimes in CI, not able to reproduce locally. For now, we have ignored the test for Android. Issue for re-enabling: https://github.com/dotnet/maui/issues/24243
```

Comment:
Did you run it using the same Api Level? We run tests in two difference API levels and sometimes, something fails from a specific version.